tests/selenium/test_bootstrap.py

```diff
@@ -0,0 +1,79 @@
+import dash
+import pytest
+import time
+
+import dash_bootstrap_components as dbc
+import dash_html_components as html
+from dash_table import DataTable
+
+import pandas as pd
+
+url = "https://github.com/plotly/datasets/raw/master/" "26k-consumer-complaints.csv"
+rawDf = pd.read_csv(url)
+df = rawDf.to_dict("rows")
+
+
+def get_app(fixed_rows, fixed_columns, ops):
+    app = dash.Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+    props = dict(
+        id="table",
+        data=df[0:250],
+        columns=[
+            {"name": i, "id": i, "hideable": i == "Complaint ID"} for i in rawDf.columns
+        ],
+        style_table=dict(height="500px", maxHeight="500px", overflow="auto"),
+        editable=True,
+        sort_action="native",
+        include_headers_on_copy_paste=True,
+        **fixed_rows,
+        **fixed_columns,
+        **ops
+    )
+
+    app.layout = html.Div([DataTable(**props)])
```

Comment:
Initially wanted this test to show a unmodified table + a another table in an iframe with bootstrap styles applied to offer some comparison point during review (e.g. only the bootstrap version changed vs. both) but the iframe content is never visible.